apps/mdn/k8s/prod/Makefile

```diff
@@ -0,0 +1,73 @@
+export PV_DB_NAME ?= pv-mdn-fast-db
+export PV_DB_CAPACITY ?= 100Gi
+export PV_DB_RECLAIM_POLICY ?= Retain
+export PV_DB_STORAGE_CLASS_NAME ?= mdn-fast-db
+export PV_DB_ARN ?= <fill-in ebs volume id from aws console>
+export PV_SHARED_NAME ?= pv-mdn-shared
+export PV_SHARED_CAPACITY ?= 1000Gi
+export PV_SHARED_RECLAIM_POLICY ?= Retain
+export PV_SHARED_STORAGE_CLASS_NAME ?= mdn-shared
+export PV_SHARED_MOUNT_PATH ?= /mdn-dev
+export PV_SHARED_ARN ?= fs-1b4cfe52.efs.us-east-1.amazonaws.com
+export MYSQL_IMAGE ?= quay.io/mozmar/mdn-mysql
+export MYSQL_IMAGE_TAG ?= latest
+export MYSQL_PERSISTENCE_SIZE ?= 100Gi
+export MYSQL_RELEASE ?= mdn
+export K8S_NAMESPACE ?= mdn-prod
+export KUMA_REPLICAS ?= 1
```

Comment:
does `KUMA_REPLICAS` control the number of pods in a K8s replica set? How is it different from `KUMA_WORKER_REPLICAS`? 

A one line comment for each would be excellent :-)